### PR TITLE
Bump csi-lib-utils to v0.9.0

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Bump csi-lib-utils to v0.9.0 to get rid of klogv1 and get health check on leader election.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
* Added leader election health check at the metrics port + path "/healthz/leader-election".
* klog/v2 is used for logging
* `metrics-address` flag is deprecated and replaced by `http-endpoint`, which enables handlers from both metrics manager and leader election health check.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Read more about how to install the example webhook [here](deploy/kubernetes/webh
 
 * `--leader-election-namespace <namespace>`: The namespace where the leader election resource exists. Defaults to the pod namespace if not set.
 
-* `--metrics-address`: The TCP network address where the prometheus metrics endpoint will run (example: `:8080` which corresponds to port 8080 on local host). The default is empty string, which means metrics endpoint is disabled.
+* `--metrics-address`: The TCP network address where the prometheus metrics endpoint and leader election health check will run (example: `:8080` which corresponds to port 8080 on local host). The default is empty string, which means metrics endpoint and leader election check are disabled.
```
Comment (msau42) [inline README.md:109]: Can you also incorporate the changes that @verult did in https://github.com/kubernetes-csi/external-attacher/pull/279?

File: cmd/csi-snapshotter/main.go [hunk 2]
```diff
@@ -135,8 +136,17 @@ func main() {
 	}
 
 	klog.V(2).Infof("CSI driver name: %q", driverName)
-	metricsManager.SetDriverName(driverName)
-	metricsManager.StartMetricsEndpoint(*metricsAddress, *metricsPath)
+
+	// Prepare http endpoint for metrics + leader election healthz
+	mux := http.NewServeMux()
+	cmm := metrics.NewCSIMetricsManagerForSidecar(driverName)
```
Comment (msau42) [inline cmd/csi-snapshotter/main.go:142]: I believe this is the correct sequence:
https://github.com/kubernetes-csi/external-attacher/pull/278/files

File: README.md (hunk 1 above)
Comment (verult) [reply README.md:109]: This contains the latest changes around the HTTP endpoint: https://github.com/kubernetes-csi/external-provisioner/pull/537

File: cmd/csi-snapshotter/main.go (hunk 2 above)
Comment (xing-yang) [reply cmd/csi-snapshotter/main.go]: Done

File: README.md (hunk 1 above)
Comment (xing-yang) [reply README.md]: Done

**Comment by msau42** [comment]
Can you also add the other points in the release note from https://github.com/kubernetes-csi/external-provisioner/pull/537?

**Comment by xing-yang** [comment]
> Can you also add the other points in the release note from kubernetes-csi/external-provisioner#537?

Sure

**Pull request merged**